server/views/partials/new_workAndSkillsPage/goals/index.njk

```diff
@@ -0,0 +1,12 @@
+{# Summary card for Goals, pulling in the relevant include #}
+
+{% from "../../../macros/summaryCardMacro.njk" import summaryCard %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{%- call summaryCard({title: "Goals", id: "goals"}) -%}
+
+    {% include './vc2GoalsOnly.njk' %}
+    {# {% include './plpGoalsOnly.njk' %} #}
```

Comment:
TODO: Logic to decide which partial to include

Reply:
Yes; it would be good to put this logic in if we can do?